src/AWS.Deploy.CLI/IFallbackCredentialsFactoryFactory.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Runtime;
+
+namespace AWS.Deploy.CLI
+{
+    /// <summary>
+    /// Represents a factory for creating AWSCredentials
```

Comment:
Replace explicit string reference of AWSCredentials to a `<see cref="AWSCredentials">`